.github/workflows/on-pullrequest.yml

```diff
@@ -115,6 +115,11 @@ jobs:
       - name: Setup Nuget.exe
         uses: nuget/setup-nuget@323ab0502cd38fdc493335025a96c8fdb0edc71f #v2.0.1
 
+      - name: Install .NET 6 runtime
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: '6.0.x'
+
```

Comment:
@dfernandez-gap we don't support .NET 6 anymore. Why are you installing it? Please tag me in the response, otherwise I might not see it.

Reply:
https://dev.azure.com/ed-fi-alliance/Ed-Fi-Alliance-OSS/_artifacts/feed/EdFi/NuGet/EdFi.Suite3.Db.Deploy/overview/3.2.2 is the version that went with ODS API v6.2 . The latest version that supports 7.x line isn't compatible with 6.x due to folder changes needed for multiple DS versions.